examples/demo33/build.gradle

```diff
@@ -50,11 +31,7 @@ dependencies {
     testImplementation project(':grails-gorm-testing-support')
 
     integrationTestImplementation "org.grails.plugins:geb"
-    integrationTestRuntimeOnly "org.seleniumhq.selenium:selenium-remote-driver:$seleniumVersion"
-    integrationTestRuntimeOnly "org.seleniumhq.selenium:selenium-api:$seleniumVersion"
-    integrationTestRuntimeOnly "org.seleniumhq.selenium:selenium-support:$seleniumVersion"
-    integrationTestRuntimeOnly "org.seleniumhq.selenium:selenium-chrome-driver:$seleniumVersion"
-    integrationTestRuntimeOnly "org.seleniumhq.selenium:selenium-firefox-driver:$seleniumVersion"
+    integrationTestImplementation testFixtures('org.grails.plugins:geb')
```

Comment:
We don't need to depend on `org.grails.plugins:geb` anymore. `JsonControllerSpec` was the only `GebSpec`

Reply:
I was torn on removing vs leaving.  I had left it since I assumed we should have more tests here.  Given that the functional test project exists, I'll remove it for now.